lib/auth/auth_test.go

```diff
@@ -44,9 +45,10 @@ import (
 func TestAPI(t *testing.T) { TestingT(t) }
 
 type AuthSuite struct {
-	bk      backend.Backend
-	a       *AuthServer
-	dataDir string
+	bk             backend.Backend
+	a              *AuthServer
+	dataDir        string
+	mockedAuditLog events.MockAuditLog
```

Comment:
Make this a pointer and initialize with `events.NewMockAuditLog`

Reply:
revised